charts/daemonset-app/values.yaml

```diff
@@ -98,23 +98,17 @@ volumes: []
 # -- A PodSpec container "startupProbe" configuration object. Note that this
 # startupProbe will be applied to the proxySidecar container instead if that
 # is enabled.
-startupProbe:
+startupProbe: {}
```

Comment:
In `simple-app` we set these to null. should we do that for consistency?

Reply:
Done